router/router_config.go

```diff
@@ -0,0 +1,6 @@
+package router
+
+type RouterConfig struct {
+	RealmConfigs  []*RealmConfig `json:"realms"`
+	RealmTemplate *RealmConfig   `json:"realm_template"`
+}
```

Comment:
moved this to `router.go`, and `RealmConfig` to `realm.go`. Alternatively, we could have created a `config.go` but I don't feel strongly at the moment either way.